src/main/java/core/utils/EnvConfiguration.java

```diff
@@ -0,0 +1,54 @@
+package core.utils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class EnvConfiguration {
+    public static Properties CONFIG = new Properties();
+
+    static {
+        try {
+            CONFIG = read();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
It should manage the exception, not only print the error.

Reply:
Exception is handled in the Properties read() method.